src/soft-play.js

```diff
@@ -6,9 +6,54 @@ let children = 0
 // the adults and children variables defined above.
 // Start with the occupancy function.
 
+function enter(numAdults, numChildren) {
+  const childEntering = numChildren >= 1
+  const childMustEnterWithAdult = numAdults >= 1
+  const multipleAdultsEntering = numAdults >= numChildren
+
+  if ((childEntering, childMustEnterWithAdult, multipleAdultsEntering)) {
```

Comment:
`true, true, false`

This isn't a valid way to construct if statements, I expect some strange and unreliable things to happen here.

You should make use of and `&&` and or `||` to construct if statements with multiple conditions. E.g.

`if (childEntering && childMustEnterWithAdult && multipleAdultsEntering)`